libbeat/docs/howto/load-index-templates.asciidoc

```diff
@@ -141,7 +107,7 @@ ifdef::no-output-logstash[]
 endif::[]
 
 ifdef::requires-sudo[]
-include::./shared-note-sudo.asciidoc[]
+include::{libbeat-dir}/shared-note-sudo.asciidoc[]
 endif::requires-sudo[]
 
 ifdef::deb_os,rpm_os[]
```

Comment:
A project for another day, but I can't wait to see the tab widget in use here as well!

Reply:
Yes! My priority is getting the quick start to show the tabs. Didn't want to do the other topics until after this PR is merged and we have the final source code for the tabbed panel in place.

Reply:
💯